di_website/events/migrations/0035_auto_20201103_1256.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1 on 2020-11-03 12:56
+
+from django.db import migrations
+
+
+def combine_time_and_location(apps, schema_editor):
+    EventPage = apps.get_model('events', 'EventPage')
+    for event in EventPage.objects.all():
+        event.time_and_location = '%s - %s %s %s' % (event.start_time.strftime("%H:%M"), event.end_time.strftime("%H:%M"), '\n', event.location)
```

Comment:
Getting this error when running migrations
![image](https://user-images.githubusercontent.com/5672438/97995164-3946c900-1df7-11eb-85ba-a717c32804b8.png)

Probably cos you're assuming `start_time` & `end_time` are always provided.

Reply:
Now checking that `start_time` and `end_time` are set before calling `strftime()`